br/pkg/task/restore_ebs_meta.go

```diff
@@ -43,18 +43,20 @@ const (
 func DefineRestoreEBSMetaFlags(command *cobra.Command) {
 	command.Flags().String(flagOutputMetaFile, "output.json", "the file path of output meta file")
 	command.Flags().Bool(flagSkipAWS, false, "don't access to aws environment if set to true")
+	command.Flags().Uint(flagCloudAPIConcurrency, defaultCloudAPIConcurrency, "concurrency of calling cloud api")
 	command.Flags().String(flagVolumeType, string(config.GP3Volume), "volume type: gp3, io1, io2")
 	command.Flags().Int64(flagVolumeIOPS, 0, "volume iops(0 means default for that volume type)")
 	command.Flags().Int64(flagVolumeThroughput, 0, "volume throughout in MiB/s(0 means default for that volume type)")
 }
 
 type RestoreEBSConfig struct {
 	Config
-	OutputFile       string               `json:"output-file"`
-	SkipAWS          bool                 `json:"skip-aws"`
-	VolumeType       config.EBSVolumeType `json:"volume-type"`
-	VolumeIOPS       int64                `json:"volume-iops"`
-	VolumeThroughput int64                `json:"volume-throughput"`
+	OutputFile          string               `json:"output-file"`
+	SkipAWS             bool                 `json:"skip-aws"`
+	CloudAPIConcurrency uint                 `json:"cloud-api-concurrency"`
```

Comment:
🤔 shall we expose this config to user?

Reply:
whole command is hidden, see https://github.com/pingcap/tidb/pull/37279/commits/93bd3219e9fc4209c13b5079e3e7084d70a5b30a

Reply:
ok, and I think we need to discuss with @IANTHEREAL if we need to expose any new config to user.